cli/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ _Released 12/3/2024 (PENDING)_
 - Removed support for Node.js 16 and Node.js 21. Addresses [#29930](https://github.com/cypress-io/cypress/issues/29930).
 - Prebuilt binaries for Linux are no longer compatible with Linux distributions based on glibc <2.28, for example: Ubuntu 14-18, RHEL 7, CentOS 7, Amazon Linux 2. Addresses [#29601](https://github.com/cypress-io/cypress/issues/29601).
 - The undocumented methods `Cypress.backend('firefox:force:gc')` and `Cypress.backend('log:memory:pressure')` were removed. Addresses [#30222](https://github.com/cypress-io/cypress/issues/30222).
+- Upgraded bundled Node.js version from `18.17.0` to `20.18.0`.
```

Comment:
Add blurb about having to navigate rather than making fetch or XHR requests in about:blank.

Reply:
```suggestion
- Upgraded bundled Node.js version from `18.17.0` to `20.18.0`.
- It is no longer possible to do things like `cy.window().then((win) => win.fetch('<some-url>')` without doing some form of initial navigation. `cy.request` will still work in this scenario.
```